src/app/core/entity-components/entity-utils/entity-utils.module.ts

```diff
@@ -0,0 +1,100 @@
+import { EditPercentageComponent } from "./dynamic-form-components/edit-percentage/edit-percentage.component";
+import { DisplayEntityComponent } from "./view-components/display-entity/display-entity.component";
+import { DisplayEntityArrayComponent } from "./view-components/display-entity-array/display-entity-array.component";
+import { DisplayTextComponent } from "./view-components/display-text/display-text.component";
+import { DisplayDateComponent } from "./view-components/display-date/display-date.component";
+import { DisplayConfigurableEnumComponent } from "./view-components/display-configurable-enum/display-configurable-enum.component";
+import { DisplayCheckmarkComponent } from "./view-components/display-checkmark/display-checkmark.component";
+import { ReadonlyFunctionComponent } from "./view-components/readonly-function/readonly-function.component";
+import { DisplayPercentageComponent } from "./view-components/display-percentage/display-percentage.component";
+import { DisplayUnitComponent } from "./view-components/display-unit/display-unit.component";
+import { MatOptionModule } from "@angular/material/core";
+import { MatSelectModule } from "@angular/material/select";
+import { ReactiveFormsModule } from "@angular/forms";
+import { ConfigurableEnumModule } from "../../configurable-enum/configurable-enum.module";
+import { MatIconModule } from "@angular/material/icon";
+import { MatTooltipModule } from "@angular/material/tooltip";
+import { MatInputModule } from "@angular/material/input";
+import { ViewModule } from "../../view/view.module";
+import { MatDatepickerModule } from "@angular/material/datepicker";
+import { MatCheckboxModule } from "@angular/material/checkbox";
+import { EntitySelectComponent } from "./entity-select/entity-select.component";
+import { MatAutocompleteModule } from "@angular/material/autocomplete";
+import { MatChipsModule } from "@angular/material/chips";
+
+@NgModule({
+  declarations: [
+    EditConfigurableEnumComponent,
+    EditTextComponent,
+    EditDateComponent,
+    EditSelectableComponent,
+    EditAgeComponent,
+    EditBooleanComponent,
+    EditLongTextComponent,
+    EditPhotoComponent,
+    EditEntityArrayComponent,
+    EditSingleEntityComponent,
+    EditPercentageComponent,
+    DisplayEntityComponent,
+    DisplayEntityArrayComponent,
+    DisplayTextComponent,
+    DisplayDateComponent,
+    DisplayConfigurableEnumComponent,
+    DisplayCheckmarkComponent,
+    ReadonlyFunctionComponent,
+    DisplayPercentageComponent,
+    DisplayUnitComponent,
+    EntitySelectComponent,
+  ],
+  imports: [
+    CommonModule,
+    MatOptionModule,
+    MatSelectModule,
+    ReactiveFormsModule,
+    ConfigurableEnumModule,
+    MatIconModule,
+    MatTooltipModule,
+    MatInputModule,
+    ViewModule,
+    MatDatepickerModule,
+    MatCheckboxModule,
+    MatAutocompleteModule,
+    MatChipsModule,
+  ],
+  entryComponents: [
+    EditConfigurableEnumComponent,
```

Comment:
These components should maybe go into the ConfigurableEnumModule.

May be worth splitting at least the more special components into their own modules also (e.g. Age, Photo or EntityArray). But not absolutely necessary at the moment, in my opinion.